src/player.cpp

```diff
@@ -3777,11 +3780,11 @@ void player::disp_status(WINDOW *w, WINDOW *w2)
             col_time = c_dkgray_red;
         }
     }
-    wprintz(w, col_time, " %3d", movecounter);
+    wprintz(w, col_time, " %d", movecounter);
 
-    wprintz(w, c_white, " %5s", _(move_mode.c_str()));
+    wprintz(w, c_white, " %s", _(move_mode == "walk" ? "W" : "R"));
```

Comment:
> did it work before

I remember there being a discussion about it. Don't remember the details, but if it got in like that I'd assume it did work. I'll change it anyway to the obvious way.

> why is the "Stm" below (and above) not translated?

Wasn't originally. Will be in a minute :)